workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css

```diff
@@ -1643,56 +1696,73 @@ pre[class*='language-'] {
 	}
 }
 
-.search-results {
-	.search-results-empty {
-		color: var(--color-neutral-8, #54555f);
-		font-weight: 400;
-	}
-
-	.search-results-entry {
-		align-items: flex-start;
-		display: flex;
-		flex-direction: column;
-		gap: 0.25rem;
-	}
-
-	.search-results-entry .search-results-entry-title {
-		border-radius: 0.625rem;
-		color: #0b5fff;
-		font-size: 1rem;
-		font-style: normal;
-		font-weight: 600 !important;
-		line-height: 1.5rem;
-		padding: 1rem;
-	}
+.learn-search-container{
+	gap: var(--spacer-5);
 
-	.search-results-entry .search-results-entry-title .published-date {
-		color: var(--color-state-neutral-darken-1, #54555F);
-		font-size: 0.8125rem;
-		font-style: normal;
-		font-weight: 400;
-		line-height: 1rem;
+	@media (max-width: 991px) {
+		flex-wrap: wrap;
 	}
 
-	.search-results-entry
-		.search-results-entry-title
-		.search-results-entry-content {
-		color: var(--color-neutral-10, #282934);
-		font-size: 1rem;
-		font-style: normal;
-		font-weight: 400;
-		line-height: 1.5rem;
-	}
+	.portlet:has( .panel-body > .learn-treeview){
+		margin-bottom: 0rem;
 
-	.search-results-entry .search-results-entry-title:hover {
-		background: var(--color-action-primary-hover-10, #edf3fe);
-	}
+		.panel-header{
+			padding: 0.75rem 1.25rem 0rem 0rem;
+		}
+	} 
 
-	.solid {
-		border-top: 1px solid var(--color-neutral-2, #e2e2e4);
+	.search-results {
+		.search-results-empty {
+			color: var(--color-neutral-8, #54555f);
+			font-weight: 400;
+		}
+	
+		.search-results-entry {
+			align-items: flex-start;
+			display: flex;
+			flex-direction: column;
+			gap: 0.25rem;
+		}
+	
+		.search-results-entry .search-results-entry-title {
+			border-radius: 0.625rem;
+			color: #0b5fff;
+			font-size: 1rem;
+			font-style: normal;
+			font-weight: 600 !important;
+			line-height: 1.5rem;
+			padding: 1rem;
+		}
+	
+		.search-results-entry .search-results-entry-title .published-date {
+			color: var(--color-state-neutral-darken-1, #54555F);
+			font-size: 0.8125rem;
+			font-style: normal;
+			font-weight: 400;
+			line-height: 1rem;
+		}
+	
+		.search-results-entry
```

Comment:
Hi @Mylena-Rodrigues 
This class 'search-results-entry' is declared twice. 

What do you think about nesting the classes ?